src/Congo.WebApi/Controllers/ProductsController.cs

```diff
@@ -14,16 +14,20 @@ namespace Congo.WebApi.Controllers
     public class ProductsController : Controller
     {
         private readonly IMediator _mediator;
+        private readonly IMapper _mapper;
 
-        public ProductsController(IMediator mediator)
+
```

Comment:
This blank line made me realize that the .editorconfig we created and added to our project wasn't included in the VS solution items, because this should be a formatting suggestion.

Remove the extra blank line for now, I'll make a quick PR to include the solution item.